hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSOutputStream.java

```diff
@@ -368,14 +368,14 @@ private DFSOutputStream(DFSClient dfsClient, String src,
     }
   }
 
-  private void adjustPacketChunkSize(HdfsFileStatus stat) throws IOException{
+  private void adjustPacketChunkSize(LocatedBlock lastBlock) throws IOException{
 
-    long usedInLastBlock = stat.getLen() % blockSize;
+    long usedInLastBlock = lastBlock.getBlockSize() % blockSize;
```

Comment:
Why don't we use `long usedInLastBlock = lastBlock.getBlockSize()` directly?

Reply:
@zhangshuyan0 Sir, Thanks for your reviewing, very nice suggestion. I will fix it.

Reply:
Have fixed.